templates/company/pscs/list.html.tx

```diff
@@ -110,6 +135,15 @@
                         % }
                     </h2>
                 % }
+
+                % if $psc.is_sanctioned {
+                  <p>
+                      <a target="_blank" href="https://www.gov.uk/government/publications/the-uk-sanctions-list">
+                          UK sanction declared
```

Comment:
I note the story said `Sanction declared`?

Reply:
thanks, updated